src/app/trivia/components/Trivia.js

```diff
@@ -1,31 +1,81 @@
-import React from 'react';
+import React, {Component} from 'react';
 import {
   Box, Title, SubBox, Option, Question, Triangle, TitleBox, BoxOption,
 } from '../styles/trivia';
+import axios from 'axios';
+import ReactLoading from 'react-loading';
 
+class Trivia extends Component {
 
-const Trivia = () => (
-  <Box>
-    <TitleBox>
-      <Title>¿Cuánto conocés?</Title>
-      <Triangle />
-    </TitleBox>
-    <SubBox>
-      <Question>¿Cuánto tarda en degradarse una botella?</Question>
-      <BoxOption>
-        <Option>10 años</Option>
-      </BoxOption>    
-      <BoxOption>
-        <Option>500 años</Option>
-      </BoxOption>   
-      <BoxOption>
-        <Option>100 años</Option>
-      </BoxOption> 
-      <BoxOption>
-        <Option>700 años</Option>
-      </BoxOption> 
-    </SubBox>
-  </Box>
-);
+  
+  constructor(props) {
+    super(props);
+    this.state = {
+      questions: [],
+      correct: 0,
+      loading: true
+    };
 
+  }
+
+  componentDidMount() {
+    this.getQuestions()
+  }
+
+  getQuestions() {
+    this.setState({loading : true})
+    axios.get('http://34.213.11.120/questions')
+      .then( ({data}) => {
+        this.setState(
+          { questions : data,
+          loading: false }
+
+        );
+      })
+      .catch((err) => {})
+  }
+
+
+
+
+render(){
+  const actualQ = this.state.questions.pop()
+  if(!this.state.loading ){ 
```

Comment:
Must use destructuring state assignment (coding style). Please define

const { loading } = this.state;

between "const actualQ" and "if" and use loading instead of this.state.loading.